brainbeats-backend/DatabaseConnection.cs

```diff
@@ -33,7 +33,7 @@ public class DatabaseConnection {
 
     public Task<ResultSet<dynamic>> ExecuteQuery(string query) {
       try {
-          return gremlinClient.SubmitAsync<dynamic>(query);
+         return gremlinClient.SubmitAsync<dynamic>(query);
       } catch (ResponseException e) {
         Console.WriteLine("\tRequest Error!");
```

Comment:
Could we have a flag that in DEBUG mode it sends the exception and in production we have generic errors? We can follow that pattern with all exceptions honestly